charts/sonarqube-dce/CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 # SonarQube Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [8.0.4]
+* Update SonarQube to 
```

Comment:
Small typo here, everything should be on same line and we are releasing `7.0.4` instead of `8.0.4`

Reply:
outch good catch! and I don't know what happened with formatting, but it is now fixed 👍